sql/src/main/java/io/crate/metadata/settings/session/SessionSettingRegistry.java

```diff
@@ -104,7 +111,18 @@
                     DataTypes.STRING.getName()
                 )
             )
-            .build();
+        .putAll(getSessionSettings())
+        .build();
+
+    private static Map<String, SessionSetting<?>> getSessionSettings() {
+        var result = new LinkedHashMap<String, SessionSetting<?>>();
```

Comment:
Would a `HashMap` also do? Or where is the stable ordering requirement coming from?